modules/osdk-suggested-namespace-node-selector.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-suggested-namespace-default-node_{context}"]
+= Setting a suggested namespace with default node selector
+
+Some Operators expect to run only on primary nodes, which can be done by setting a `nodeSelector` in the Pod spec by the Operator itself.
+
+To avoid getting a duplicated and potential conflicting cluster-wide default `nodeSelector`, you can set a default node selector on the namespace where the Operator runs. The default node selector will take precedence over the cluster default so the cluster default will not be applied to the pods in the Operators namespace.
+
+When adding the Operator to a cluster using OperatorHub, the web console auto-populates the suggested namespace for the cluster administrator during the installation process, and the suggested namespace is created directly using the namespace manifest in YAML included in the cluster service version (CSV).
+
+.Procedure
+
+* In your CSV, set the `operatorframework.io/suggested-namespace-template` with a manifest for a Namespace object. The following sample is a manifest for an example Namespace with the namespace default node selector specified:
++
+[source,yaml]
+----
+metadata:
+  annotations:
+    operatorframework.io/suggested-namespace-template: <1>
+      {
+        "apiVersion": "v1",
+        "kind": "Namespace",
+        "metadata": {
+          "name": "vertical-pod-autoscaler-suggested-template",
+          "annotations": {
+            "openshift.io/node-selector": ""
+          }
+        }
+      }
```

Comment:
Hi @opayne1,  I would like to suggest adding a note which is `If both suggested-namespace and suggested-namespace-template annotation are present in CSV template should take precedence.`

This means if both annotations are present, the value set in the "suggested-namespace-template" will be the default value displayed on the operator-recommended namespace on OCP Console page

Reply:
Updated with the note. Thank you!